lib/utils/yaml-utils.ts

```diff
@@ -35,6 +35,15 @@ export function readYamlDocument(path: string): string {
     }
 }
 
+export function loadMultiResourceYaml(path: string): any {
```

Comment:
Could we add `jsdoc` to say what these new functions are doing?

Reply:
added jsdocs on all yaml helper functions.